ibm_catalog.json

```diff
@@ -0,0 +1,546 @@
+            {
+              "key": "kms_encryption_enabled_boot_volume"
+            },
+            {
+              "key": "existing_kms_instance_crn"
+            },
+            {
+              "key": "skip_block_storage_kms_iam_auth_policy"
+            },
+            {
+              "key": "kms_endpoint_type",
+              "options": [
+                {
+                  "displayname": "Public",
+                  "value": "public"
+                },
+                {
+                  "displayname": "Private",
+                  "value": "private"
+                }
+              ]
+            },
+            {
+              "key": "boot_volume_key_ring_name"
+            },
+            {
+              "key": "existing_boot_volume_kms_key_crn"
+            },
+            {
+              "key": "force_delete_kms_key"
+            },
+            {
+              "key": "use_boot_volume_key_as_default"
+            },
+            {
+              "key": "security_group"
+            },
+            {
+              "key": "security_group_ids"
+            },
+            {
+              "key": "boot_volume_key_name"
+            },
+            {
+              "key": "boot_volume_snapshot_id"
+            },
+            {
+              "key": "dedicated_host_id"
+            },
+            {
+              "key": "enable_floating_ip"
+            },
+            {
+              "key": "load_balancers"
+            },
+            {
+              "key": "manage_reserved_ips"
+            },
+            {
+              "key": "placement_group_id"
+            },
+            {
+              "key": "primary_virtual_network_interface_additional_ip_count"
+            },
+            {
+              "key": "secondary_allow_ip_spoofing"
+            },
+            {
+              "key": "secondary_floating_ips"
+            },
+            {
+              "key": "secondary_security_groups"
+            },
+            {
+              "key": "existing_secondary_subnet_id",
+              "custom_config": {
+                "type": "platform_resource",
+                "grouping": "deployment",
+                "original_grouping": "deployment",
+                "config_constraints": {
+                  "resourceType": "subnet",
+                  "selection": "single_select",
+                  "valueType": "id"
+                },
+                "associations": {
+                  "parameters": [
+                    {
+                      "name": "region",
+                      "optionsRefresh": true
+                    }
+                  ]
+                }
+              }
+            },
+            {
+              "key": "secondary_use_vsi_security_group"
+            },
+            {
+              "key": "snapshot_consistency_group_id"
+            },
+            {
+              "key": "use_static_boot_volume_name"
+            },
+            {
+              "key": "user_data"
```

Comment:
should this be a multi line input like you mentioned in the variable description?